### PR TITLE
[Improvement] ipcRenderer.SendSync to ipcRenderer.on/send

### PR DESCRIPTION
**Pull request by evertonlperes**
### Changes
1. Replaced remaining `ipcRenderer.SendSync('settings-read')` in vue files to `ipcRenderer.on/send('settings-read')`
2. Replaced `get-app-version` handler to `send() --> event.reply()` on `background.ts`
3. Replaced `settings-read` handler to `send() --> event.reply()` on `background.ts`

Motivator: https://github.com/rancher-sandbox/rancher-desktop/pull/855

Note: As @ericpromislow had recommended, avoiding/reducing the `ipcRenderer.invoke()` usage and replacing it to a combination of `ipcRenderer.on()` and `ipcRenderer.send()`.
The app does not showed any issue and it is performing properly.

~~Our E2E tests was really intermittent and it was crashing during the `SendSync` event. As our E2E perform on an `async` way, replacing it to `.invoke` have mitigated the flaky/intermittence caused by `SendSync` method.
Here's a reference to replace `SendSync` to `invoke`: 
Screen Shot 2021-11-12 at 3 48 55 PM
Ref: [ipcRenderer.SendSync()](https://www.electronjs.org/docs/latest/api/ipc-renderer#ipcrenderersendsyncchannel-args)~~

### CODE REVIEW

**Review by mook-as — APPROVED** [review]

**Comment by evertonlperes** [comment]
Just fixed a merge conflict on the last push.
Could you guys take a look again? @mook-as @ericpromislow ?

Thanks!

File: src/pages/K8s.vue [hunk 1]
```diff
@@ -201,6 +201,14 @@ export default {
       console.log('settings have been updated');
       this.$data.settings = settings;
     });
+    (async() => {
```
Comment (ericpromislow) [inline src/pages/K8s.vue:204]: I think we should minimize our uses of `ipcRenderer.invoke` in the UI, and instead break it into a `send` and an `on`, like the use of `images-namespaces` and `images-namespaces-read` at https://github.com/rancher-sandbox/rancher-desktop/blob/f3d631a161ac684983b81329781ce5c44ddc9acf/src/pages/Images.vue#L95  (but please wait for Mark's opinion).

File: src/pages/PortForwarding.vue [hunk 2]
```diff
@@ -47,6 +48,13 @@ export default {
       // TODO: put in a status bar
       this.$data.settings = settings;
     });
+    (async() => {
```
Comment (ericpromislow) [inline src/pages/PortForwarding.vue:51]: Same comment as above.

File: src/pages/K8s.vue [hunk 3]
```diff
@@ -88,7 +88,7 @@ export default {
       currentEngine:        ContainerEngine.NONE,
       containerEngineNames: ContainerEngineNames,
       /** @type Settings */
-      settings:             ipcRenderer.sendSync('settings-read'),
+      settings:             defaultSettings,
```
Comment (ericpromislow) [inline src/pages/K8s.vue:91]: Initing to `defaultSettings` is good.

**Review by ericpromislow — CHANGES_REQUESTED** [review]

**Comment by ericpromislow** [comment]
Merge conflicts in FirstRun.vue -- please rebase and push

**Review by ericpromislow — APPROVED** [review]
LGTM

**Pull request merged**